CMakeLists.txt

```diff
@@ -432,8 +432,11 @@ endif()
 
 if(EMSCRIPTEN)
     # Ensure required SDL2 ports (e.g. SDL2) are built before header preloading
+    if (CMAKE_C_COMPILER MATCHES ".bat")
```

Comment:
Is this a better way to detect the if the compiler is running on windows: https://github.com/emscripten-core/emscripten/issues/3191